spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/Topology.scala

```diff
@@ -28,7 +29,11 @@ import scala.reflect.ClassTag
 
 class Model[T: ClassTag](private val _inputs : Seq[ModuleNode[T]],
       private val _outputs : Seq[ModuleNode[T]])(implicit ev: TensorNumeric[T])
-  extends StaticGraph[T](_inputs, _outputs, None, false) {
+  extends StaticGraph[T](_inputs, _outputs, None, false) with Training[T] {
+
+  override var model: Module[T] = this
```

Comment:
It's weird to add the `model` here.

Reply:
This is to override the model for compile and fit. Any suggestion for modification?